tox.ini

```diff
@@ -25,9 +25,7 @@ commands =
     {toxinidir}/test {posargs}
 
 [testenv:latest-pip]
-whitelist_externals = sed
 commands =
-    sed -i 's/,<=9.999//g' {toxinidir}/setup.py {toxinidir}/requirements.d/import_tests.txt
```

Comment:
this should probably still happen -- we should be upper-bounding the pip version since it's likely to break

Reply:
From my understanding the sed command was removing the 9.999 upper-bound from setup.py.

At the moment there is only a lower-bound to pip>=10.0.0 https://github.com/Yelp/venv-update/blob/056c652d6af0a6ed9671570908b57f59fb68094e/setup.py#L58

do you prefer me keeping this line so once an upper bound will be defined it will be just a matter of changing the number?

Reply:
I'm suggesting adding an upper bound and keeping the sed